### ISSUE TITLE
Get Name of device in a group with : Get-MgGroupMember 

### ISSUE THREAD

**Issue by kornflex** [microsoftgraph/msgraph-sdk-powershell#525]
Hello,

I'm trying to get all devices in a group :

Get-MgGroupMember -GroupId $GroupId | Select-Object  *

But I just get this fields : 
deleteddatetime, id, keys, values, additionalproperties and count.

How can I get the name of each device ?

Thank you very much 
[AB#7842](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7842)

**Comment by peombwa** [comment]
This is dependent on #423 and #19.  We currently don't have a way of [casting OData OpenTypes](https://docs.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0&tabs=http#example-3-use-odata-cast-to-get-only-a-count-of-user-membership) to entities, but we are exploring various options of supporting this in PowerShell.

A workaround for you would be to use the properties stored in `AdditionalProperties` like this:
``` powershell
Get-MgGroupMember -GroupId $GroupId | Foreach-Object {
    if ($_.AdditionalProperties["@odata.type"] -eq "#microsoft.graph.device"){
        Write-Host $_.AdditionalProperties.displayName    
    }
}
```

**Comment by peombwa** [comment]
This also affects `Get-MgDirectoryObject`, see https://docs.microsoft.com/en-us/graph/api/directory-deleteditems-list?view=graph-rest-1.0#examples-using-the-orderby-odata-query-parameter.

**Comment by peombwa** [comment]
Closing as a duplicate of https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/19.
